examples/asset/processing/processing.rs

```diff
@@ -17,18 +17,18 @@ use thiserror::Error;
 
 fn main() {
     App::new()
-        // Enabling `processed_dev` will configure the AssetPlugin to use asset processing.
-        // This will run the AssetProcessor in the background, which will listen for changes to
-        // the `assets` folder, run them through configured asset processors, and write the results
-        // to the `imported_assets` folder.
+        // Using the "processed" mode will configure the AssetPlugin to use asset processing.
+        // If you also enable the `asset_processor` cargo feature, this will run the AssetProcessor
+        // in the background, which will listen for changes to the `assets` folder, run them through
```

Comment:
this makes it seems that the `asset_processor` feature is responsible for file watching